src/models/Genre/GenreModel.ts

```diff
@@ -7,7 +8,7 @@ export const GENRE_CONTENT_TYPE = 'genre';
 export type Genre = {
 	id: string;
 	title: string | null;
-	movies: Array<GenreMovie>;
+	movies: Array<GenreMovie> | Array<MovieModel>;
```

Comment:
Since we have a MovieModel now, we can get rid of the `GenreMovie` and just use `Array<Partial<MovieModel>>`